barcampgb.org

```diff
@@ -0,0 +1 @@
+Subproject commit 8ae7da9b8bfbfc64e32d7d83f9d693bc1d606f42
```

Comment:
I believe it is done

On Thu, Jun 2, 2016 at 7:39 AM Chris Jaure <notifications@github.com> wrote:

> In barcampgb.org
> <https://github.com/BarCampGreenBay/barcampgb.org/pull/51#discussion_r65531253>
> :
>
> > @@ -0,0 +1 @@
> > +Subproject commit 8ae7da9b8bfbfc64e32d7d83f9d693bc1d606f42
>
> Yeah you need to stage and commit and push. Staging deleted files can be
> tricky. git add -u will be helpful.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/BarCampGreenBay/barcampgb.org/pull/51/files/f253472d73b2761f72c09276d0a07c3b2aea4f8a#r65531253>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AIjw8Y91Uxyb5jEnUU5y0G_nMuK7kUJnks5qHs8WgaJpZM4IrILu>
> .
>
-- 
Sent from my iPhone